src/ripple/net/impl/RPCCall.cpp

```diff
@@ -1353,6 +1354,20 @@ rpcCmdLineToJson (std::vector<std::string> const& args,
 
     jvRequest   = rpParser.parseCommand (args[0], jvRpcParams, true);
 
+    auto insert_api_version = [](Json::Value & jr){
+        if(!jr.isMember(jss::error) && !jr.isMember(jss::api_version))
+            jr[jss::api_version] = RPC::ApiMaximumSupportedVersion;
+    };
+
+    if(jvRequest.isObject())
+        insert_api_version(jvRequest);
+    else if(jvRequest.isArray()) {
+        for (Json::UInt j = 0; j < jvRequest.size(); ++j) {
```

Comment:
Thanks, I changed to `std::for_each(jvRequest.begin(), jvRequest.end(), insert_api_version);`

I modified `insert_api_version()` to `include isObject()`. I think that is still needed. Otherwise, `Value::operator[]` could be called on a non-object.